<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tableau">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>1141</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion des promotions</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1131</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Ajouter une promotion</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_reference">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">/*background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);*/

border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_pourcentage">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">/*background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);*/

border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>200</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Pourcentage</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>250</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>date de debut</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>300</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Dernier d√©lai </string>
        </property>
       </widget>
       <widget class="QPushButton" name="Ajouter_Button">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>450</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>200</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Pourcentage</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>-20</x>
          <y>0</y>
          <width>1131</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image.jpg);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>150</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>reference du produit</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="lineEdit_debut">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>260</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="lineEdit_fin">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>300</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <zorder>graphicsView</zorder>
       <zorder>label_9</zorder>
       <zorder>label_17</zorder>
       <zorder>lineEdit_reference</zorder>
       <zorder>label_11</zorder>
       <zorder>Ajouter_Button</zorder>
       <zorder>lineEdit_pourcentage</zorder>
       <zorder>label_10</zorder>
       <zorder>label_19</zorder>
       <zorder>lineEdit_debut</zorder>
       <zorder>lineEdit_fin</zorder>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Supprimer une promotion</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>La reference</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_supp">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>80</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">/*background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);*/

border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Supp_Button">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>260</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1131</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image.jpg);</string>
        </property>
       </widget>
       <zorder>graphicsView_2</zorder>
       <zorder>label_12</zorder>
       <zorder>lineEdit_supp</zorder>
       <zorder>Supp_Button</zorder>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Afficher les promotions</string>
       </attribute>
       <widget class="QTableView" name="tabpromotions">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>841</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 46), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1131</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image.jpg);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_supp_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>60</y>
          <width>411</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">/*background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);*/

border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>La reference</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>500</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Trier</string>
        </property>
       </widget>
       <zorder>graphicsView_3</zorder>
       <zorder>tabpromotions</zorder>
       <zorder>lineEdit_supp_2</zorder>
       <zorder>label_16</zorder>
       <zorder>pushButton</zorder>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Modifier une promotion</string>
       </attribute>
       <widget class="QLineEdit" name="Modif_ref">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>80</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">/*background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);*/

border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>180</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>date de debut</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>230</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Dernier d√©lai </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="modif_pour">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">/*background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);*/

border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Reference du produit</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Pourcentage</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1131</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image.jpg);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Modifier">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>380</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="modif_debut">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>180</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="modif_fin">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>220</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <zorder>graphicsView_4</zorder>
       <zorder>Modif_ref</zorder>
       <zorder>label_13</zorder>
       <zorder>label_14</zorder>
       <zorder>modif_pour</zorder>
       <zorder>label_8</zorder>
       <zorder>label_15</zorder>
       <zorder>Modifier</zorder>
       <zorder>modif_debut</zorder>
       <zorder>modif_fin</zorder>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2">
     <attribute name="title">
      <string>Gestion</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1181</width>
      <height>651</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_pic</zorder>
   <zorder>Tableau</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
